src/Conference/__snapshots__/ConferenceDetails.test.js.snap

```diff
@@ -1,27 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ConferenceDetails renders conference details 1`] = `
```

Comment:
I don't think you should've updated this snapshot as its now not showing the conference details! I think you should perhaps try to mock the request and resolve the promise so it sets some data in the state and renders the component as opposed to just the loading state

Reply:
We are a bit unsure on how to proceed with this, could you offer any further explanation? Thanks!